docs/topics/native/native-overview.md

```diff
@@ -14,14 +14,16 @@ self-contained program that does not require an additional runtime or virtual ma
 ## Target platforms
 
 Kotlin/Native supports the following platforms:
-   * iOS (arm32, arm64, simulator x86_64)
-   * macOS (x86_64)
-   * watchOS (arm32, arm64, x86, x86_64)
-   * tvOS (arm64, x86_64)
-   * Android (arm32, arm64, x86, x86_64)
-   * Windows (mingw x86_64, x86)
-   * Linux (x86_64, arm32, arm64, MIPS, MIPS little endian)
-   * WebAssembly (wasm32)
+   * iOS
+   * macOS
+   * watchOS
+   * tvOS
```

Comment:
Maybe consolidate iOS, watchOS and tvOS into a single bullet point?

Reply:
Makes sense.